### DISCUSSION TITLE
Is PowerGrid compatible with Mary UI?

### DISCUSSION THREAD

**Discussion by reachfirst7054** [Power-Components/livewire-powergrid#1323] [Q&A] [answered]
I was trying to use the PowerGrid component with MaryUI. It is not working due to table input filters. Am I doing something wrong or it is not compatible?

**Comment by luanfreitasdev** [comment]
Hello!. Can you provide an example and what errors are occurring?

**Comment by pboese** [comment, marked as answer]
I'm using PowerGrid with MaryUI and it works
